HISTORY.md

```diff
@@ -17,6 +17,7 @@
 * Fix a race condition between WritePrepared::Get and ::Put with duplicate keys.
 * Fix crash when memtable prefix bloom is enabled and read/write a key out of domain of prefix extractor.
 * Adjust WriteBufferManager's dummy entry size to block cache from 1MB to 256KB.
+* Fix a bug in GetOverlappingInputsRangeBinarySearch where a pointer should be dereferenced.
```

Comment:
It needs to be clearer. Right now it is not very helpful to users as a part of the release note.

Also, it is under release 6.1.1, which claims to be cut in 4/9/2019. Something must be wrong somewhere.